### ISSUE TITLE
Diagnostics should include agent ID

### ISSUE THREAD

**Issue by joshdover** [elastic/elastic-agent#190]
When running `elastic-agent diagnostics collect` the Agent's ID should be included somewhere in the report. This is helpful for SDHs to help users with identifying data coming from this given Agent. However, today (I'm using Agent 8.1.0) the `config/elastic-agent-local.yaml` file has an empty string for the agent.id field, for example:

```yaml
fleet:
  enabled: true
  access_api_key: XXXXX
  protocol: http
  host: XXXXX
  hosts:
  - XXXXX
  ssl:
    verification_mode: full
    renegotiation: never
  timeout: 10m0s
  reporting:
    threshold: 10000
    check_frequency_sec: 30
  agent:
    id: "" # this is empty!
```

**Comment by ph** [comment]
@michel-laterman If you do any changes in the diagnostic, this seems a low-hanging-fruit to me.
\\

**Comment by michel-laterman** [comment]
yeah, this should be pretty easy

**Comment by amolnater-qasource** [comment]
Hi Team
We have revalidated this issue and found it fixed on latest 8.3 Snapshot Kibana cloud environment.
Detailed observations are shared on related PR at https://github.com/elastic/elastic-agent/pull/262

**Build details:**
BUILD: 52512
COMMIT: df225b213b188c81888141cee2ec191424fc0649

Hence marking this as QA:Validated.
Thanks
